omero/sysadmins/limitations.txt

```diff
@@ -18,7 +18,10 @@ the current Ubuntu build of 8u40) have broken JPEG support, reported
 for `OpenJDK
 <http://icedtea.classpath.org/bugzilla/show_bug.cgi?id=1393>`__ and
 `Ubuntu
-<https://bugs.launchpad.net/ubuntu/+source/openjdk-8/+bug/1421501>`__.
+<https://bugs.launchpad.net/ubuntu/+source/openjdk-8/+bug/1421501>`__. See
+:ticket:`ticket 12612` and
+:mailinglist:`this ome-users thread <ome-users/2015-November/005723.html>` for
```

Comment:
this seems to link to the pipermail URL instead of the mailman/listinfo one that you want